File: Assignment6/Assignment6_5.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment6_5.c 
// Description : Write a program which accept number from user and print table reverse.
// Author      : Aniket Devaba Markad 
// Date        : 16/05/2025
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

#include<stdio.h>

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 
// 
// Function Name : DisplayTableRev
// Description   : This function is used to print table reverse.
// Input         : int 
// Output        : void
// Author        : Aniket Devaba Markad 
// Date          : 16/05/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

void DisplayTable(int iNo)
{
   int iCnt = 0;
   int iTable = 1;

   if(iNo < 0)
   {
        iNo = -iNo;
   }

   for(iCnt = 10;iCnt >= 1; iCnt--)
   {
        iTable = iNo * iCnt;
        printf("%d\n",iTable);
   }

}

int main()
{
    int  iValue = 0;
    
    printf("Enter number :");
    scanf("%d ",&iValue);

    DisplayTable(iValue);

    return 0;
}
//Time Complexcity is : O(n)

File: Java/Assignment6_5.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment6_5.java
// Description : Accept number from user and print table in reverse order
// Author      : Aniket Devaba Markad                        
// Date        : 1/1/2026
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Number
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : DisplayTableRev
// Description   : This function is used to print table in reverse . 
// Author        : Aniket Devaba Markad 
// Date          : 1/1/2026 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

    void DisplayTableRev(int iNo)
    {
        int itemp = 1;

        for(int i = 10; i >= 1;i--)
        {
            itemp = iNo * i;
            System.out.println(itemp);
        }
    }
}
public class Assignment6_5
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter the number ");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        nobj.DisplayTableRev(iValue);

    }
}
